client/src/components/ActionColumn.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Table } from "semantic-ui-react";
+import { Link } from "react-router-dom";
+
+const ActionColumn = props => {
+  const { id, status, translatorName, userName, userRole } = props;
+  if (userRole === "User") {
+    return (
+      <Table.Cell>
+        <span>
+          <Link to={`/document/${id}`}>View</Link>
+        </span>{" "}
+        / <span>Delete</span>
+      </Table.Cell>
+    );
+  } else if (userName === translatorName && status === "Processing") {
+    return (
+      <Table.Cell>
+        <span>
+          <Link to={`/document/${id}`}>View</Link>
+        </span>{" "}
+        /{" "}
+        <Link to={`/add-document-translation/${id}`}>
+          <span>Submit Translation</span>
+        </Link>
+      </Table.Cell>
+    );
+  } else {
+    return (
+      <Table.Cell>
+        <span>
+          <Link to={`/document/${id}`}>View</Link>
+        </span>{" "}
+        / <span>Pick Translation</span>
```

Comment:
Pick translation should be hidden if the status is not waiting. Currently, this would be displayed when the status is completed also.

Reply:
Done